litellm/llms/sagemaker.py

```diff
@@ -185,7 +182,8 @@ def completion(
         # I assume majority of users use .env for auth
         region_name = (
             get_secret("AWS_REGION_NAME")
-            or "us-west-2"  # default to us-west-2 if user not specified
+            or aws_region_name  # get region from config file if specified
```

Comment:
this looks like the key chane if i'm not mistaken

Reply:
Could you help me understand what you mean? I'm not sure I follow. From testing/tests, I can show this:
```
% python
Python 3.10.5 (v3.10.5:f377153967, Jun  6 2022, 12:36:10) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> a=None
>>> b=None
>>> c="us-west-2"
>>> a or b or c
'us-west-2'
>>> b="us-east-1"
>>> a or b or c
'us-east-1'
>>> a="us-east-2"
>>> a or b or c
'us-east-2'
```